wp-includes/rest-api/auth/class-wp-rest-token.php

```diff
@@ -762,6 +762,14 @@ public function get_auth_header() {
 			$header = sanitize_text_field( $_SERVER['REDIRECT_HTTP_AUTHORIZATION'] );
 		}
 
+		//Maybe apache?
```

Comment:
This whole block of code does not adhere to coding standards as is causing phpcs to fail the build. Please update it when you have a chance.

Reply:
Perhaps it would make more sense to check for the missing header at the same time as the check for the function.

```
// Fallback for Apache request headers.
if ( ! $header && function_exists( 'apache_request_headers' ) ) {
    $headers = apache_request_headers();
    if ( is_array( $headers ) && ! empty( $headers[ 'Authorization' ] ) ) {
        $header = sanitize_text_field( $headers[ 'Authorization' ] );
    }
}
```

Reply:
This doesn't have anything to do with Multisite though. Can you describe the problem you're having with Multisite so I can understand why you are changing the rest endpoint?